sweep/sweeper.go

```diff
@@ -1441,11 +1441,6 @@ func (s *UtxoSweeper) markInputFailed(pi *SweeperInput, err error) {
 
 	pi.state = Failed
 
-	// Remove all other inputs in this exclusive group.
-	if pi.params.ExclusiveGroup != nil {
-		s.removeExclusiveGroup(*pi.params.ExclusiveGroup)
-	}
```

Comment:
do we want to still mark this input as "Excluded" though or no?

Reply:
actually - seems to make sense just to leave it as failed!

Reply:
agree let's keep it failed.